src/analysis/special_functions/polynomials.lean

```diff
@@ -0,0 +1,137 @@
+/-
+Copyright (c) 2020 Anatole Dedecker. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anatole Dedecker
+-/
+import analysis.asymptotic_equivalent
+import data.polynomial.erase_lead
+
+/-!
+# Limits related to polynomial and rational functions
+
+This file proves basic facts about limits of polynomial and rationals functions.
+The main result is `eval_is_equivalent_at_top_eval_lead`, which states that for
+any polynomial `P` of degree `n` with leading coefficient `a`, the corresponding
+polynomial function is equivalent to `a * x^n` as `x` goes to +∞.
+
+We can then use this result to prove various limits for polynomial and rational
+functions, depending on the degrees and leading coefficients of the considered
+polynomials.
+-/
+
+open filter finset asymptotics
+open_locale asymptotics topological_space
+
+namespace polynomial
+
+variables {α : Type*} [normed_linear_ordered_field α] [order_topology α]
```

Comment:
Could you rename your field to `𝕜` (typed with `\bbk`) in the whole file? This is the convention for normed fields in the analysis folder.